### PR TITLE
Adding GIF videos for example folder

### PR DESCRIPTION
**Pull request by Roderick-Jonsson**
This will help new visitors to check out the result of the examples without having to download the repo and start running code.

Used [GIPHY Capture](https://giphy.com/apps/giphycapture) for creating Gif's as other tools didn't seem to work on Mac OS X.

### CODE REVIEW

**Review by servusdei2018 — APPROVED** [review]

**Comment by khatibomar** [comment]
Hi I was looking for a show case for list-fancy
Maybe you forget to include it , I don't know about other examples.

**Comment by Roderick-Jonsson** [comment]
@khatibomar  I am not able to run the list-fancy example as it gives me errors
```
$ go run list-fancy/main.go 
# command-line-arguments
list-fancy/main.go:77:17: undefined: randomItemGenerator
list-fancy/main.go:79:17: undefined: delegateKeyMap
list-fancy/main.go:84:17: undefined: randomItemGenerator
list-fancy/main.go:85:19: undefined: newDelegateKeyMap
list-fancy/main.go:97:14: undefined: newItemDelegate
```
cc: @muesli

**Comment by muesli** [comment]
> @khatibomar I am not able to run the list-fancy example as it gives me errors

This example consists of more than one Go file. Either `go build` it or run it with: `go run list-fancy/*.go`.

**Comment by Roderick-Jonsson** [comment]
@khatibomar Added your missing list fancy gif :)

**Comment by meowgorithm** [comment]
Thanks for the PR @Roderick-Jonsson — this is really nice.

My hesitations with this one is clones becoming slower because of the extra file size as well as having to keeping the GIFs up to date (the Glamour one is outdated now, in fact).

In the first case we could potentially move the GIFs elsewhere (most if the ones in our READMEs are on stuff.charm.sh), though it would make contributions more difficult.

File: examples/textinput/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# textinput Example
+
+![textinput Recording](recording-textinput.gif)
```
Comment (winder) [inline examples/textinput/README.md:3]: There is a typo in the filename, recording-texti**m**put.gif

File: examples/textinputs/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# textinputs Example
+
+![textinputs Recording](recording-textinputs.gif)
```
Comment (winder) [inline examples/textinputs/README.md:3]: This one also, texti**m**puts.

**Comment by winder** [comment]
@meowgorithm these gifs would have been convenient when I was comparing bubbletea and gocui. Hosting them elsewhere seems like a good solution.

**Comment by bashbunni** [comment]
These are awesome! I think we should be running them with `go run .` or `go build` as muesli suggested so other users don't run into issues. If you want me to add the files to `stuff.charm.sh` I can do that once they've been updated. 

I also see the issue with these becoming quickly outdated. @meowgorithm maybe we can take on two birds with one stone by using these 🧋 examples as content, so we would have recordings of them that we can include in the repo and reuse for IG, YouTube, Twitter, etc. That would make it easier to keep them up-to-date. Thoughts?

**Comment by meowgorithm** [comment]
Hey everyone. We discussed this some more and we're going to forgo this one for now simply to mitigate the additional maintenance this will incur (some of the GIFs, in fact, are already out of date). Many of the examples shown in this repo can also be seen in some fashion in the [Bubbles Repo](https://github.com/charmbracelet/bubbles).

@Roderick-Jonsson, thank you for all the effort and attention to detail in this PR. We genuinely appreciate the effort.

As a side note, it _would_ be nice to figure out a way to automate the GIF generation at some point.

**Pull request closed without merging**